Source/x11/XGServerWindow.m

```diff
@@ -4177,29 +4177,32 @@ - (void) standardcursor: (int)style : (void **)cid
     case GSIBeamCursor:
       cursor = XCreateFontCursor(dpy, XC_xterm);
       break;
-    case GSCrosshairCursor:
-      cursor = XCreateFontCursor(dpy, XC_crosshair);
+    case GSClosedHandCursor:
+      cursor = XCreateFontCursor(dpy, XC_fleur);
```

Comment:
Why use fleur for closed hand?

Reply:
Because XC_flue is the closest item to the closed hand. Anyway if you want to have closed hand cursor image you should use [NSCursor closedHandCursor]. It loads the right image (see appropriate PR in libs-gui). Probably these 2 lines hould be removed. What do you think?

Reply:
I agree, removing these lines is less confusing.

Reply:
Done.